src/Aspire.Dashboard/Components/Controls/SpanDetails.razor

```diff
@@ -96,20 +97,18 @@
                         @FilteredSpanLinks.Count()
                     </FluentBadge>
                 </div>
-                <FluentDataGrid ResizeLabel="@AspireFluentDataGridHeaderCell.GetResizeLabel(Loc)"
-                                ResizeType="DataGridResizeType.Discrete"
```

Comment:
What consequences come from removing these?

Reply:
There we're needed. This grid doesn't have resizing or sorting.